hivemind/client/averaging/__init__.py

```diff
@@ -75,7 +75,7 @@ class DecentralizedAverager(mp.Process, averaging_pb2_grpc.DecentralizedAveragin
           local tensors for averaging
     :param allow_state_sharing: if set to True, other peers can download this peer's state. Can be overwritten
       with averager.allow_state_sharing = True / False
-
+    :param shutdown_grace_seconds: when calling .shutdown, wait for up to this many seconds before terminating
```

Comment:
nit: This message made me think that the server always waits for `shutdown_grace_seconds` shutdown. I suggest to rename the parameter to `shutdown_grace_timeout` and clarify the message.